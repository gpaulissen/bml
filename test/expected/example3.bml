#+TITLE: 

#+DESCRIPTION: 

#VUL 00

#SEAT 0

1C
    1D
        1H = Minimum with 2--3!h
            1S = 4+!h, 4!s, at most invitational
            1N = Sign off
            2C = Puppet to 2!d
                2D = Forced
                    2H = Mildly invitational with 5!h
                    2S = Invitational, 5+!h and 4!s
                    2N = Strongly invitational with 5!h
                    3C = Invitational with 4!h and 5+!c
                    3D = Invitational with 4!h and 5+!d
                    3H = 6!h, about 11--12 hcp
            2D = Artificial game force
            2H = 6+!h, about 9--10 hcp
        1S = 5+!c, 4+!s, unlimited
        1N = 17--19 bal, 2--3!h
        2C = 5+!c, unbal, 0--2!h, 0--3!s
        2D = Reverse
        2H = Minimum, 4!h
        2S = 16+ hcp, 5+!c and 4+!h
            3D = Retransfer
                3H
                    3S = Cue bid, slam interest
                    4C = Cue bid, slam interest
                    4D = Cue bid, slam interest
                    4H = To play
            3H = Invitational
            3S = Splinter
            4C = Splinter
            4D = Splinter
            4H = To play
        2N = 16+ hcp, 6+!c. 18+ if 3!h
            3C = Suggestion to play
            3D = Relay
                3H = 3!h, 18+ hcp
            3H = Game forcing with 6+!h
        3C = 15--17 hcp, 6+!c and 3!h
            3D = Retransfer
            3H = Invitational
        3D = 17--19 bal, 4!h
            3H = To play
        3H = 13--15 hcp, good hand, 5+!c and 4!h
            3N = Asking for singleton
