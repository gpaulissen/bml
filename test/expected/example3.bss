*00{}=NYYYYYY
001C=NYYYYYY008
001CP1D=NYYYYYY008
001CP1DP1H=NYYYYYY008Minimum with 2--3!h
001CP1DP1HP1S=NYYYYYY0084+!h, 4!s, at most invitational
001CP1DP1HP1N=NYYYYYY0Sign off
001CP1DP1HP2C=NYYYYYY008Puppet to 2!d
001CP1DP1HP2CP2D=NYYYYYY008Forced
001CP1DP1HP2CP2DP2H=NYYYYYY008Mildly invitational with 5!h
001CP1DP1HP2CP2DP2S=NYYYYYY008Invitational, 5+!h and 4!s
001CP1DP1HP2CP2DP2N=NYYYYYY0Strongly invitational with 5!h
001CP1DP1HP2CP2DP3C=NYYYYYY008Invitational with 4!h and 5+ minor
001CP1DP1HP2CP2DP3D=NYYYYYY008Invitational with 4!h and 5+ minor
001CP1DP1HP2CP2DP3H=NYYYYYY0086!h, about 11--12 hcp
001CP1DP1HP2D=NYYYYYY008Artificial game force
001CP1DP1HP2H=NYYYYYY0086+!h, about 9--10 hcp
001CP1DP1S=NYYYYYY0085+!c, 4+!s, unlimited
001CP1DP1N=NYYYYYY017--19 bal, 2--3!h
001CP1DP2C=NYYYYYY0085+!c, unbal, 0--2!h, 0--3!s
001CP1DP2D=NYYYYYY008Reverse
001CP1DP2H=NYYYYYY008Minimum, 4!h
001CP1DP2S=NYYYYYY00816+ hcp, 5+!c and 4+!h
001CP1DP2SP3D=NYYYYYY008Retransfer
001CP1DP2SP3DP3H=NYYYYYY008
001CP1DP2SP3DP3HP3S=NYYYYYY008Cue bid, slam interest
001CP1DP2SP3DP3HP4C=NYYYYYY008Cue bid, slam interest
001CP1DP2SP3DP3HP4D=NYYYYYY008Cue bid, slam interest
001CP1DP2SP3DP3HP4H=NYYYYYY008To play
001CP1DP2SP3H=NYYYYYY008Invitational
001CP1DP2SP3S=NYYYYYY008Splinter
001CP1DP2SP4C=NYYYYYY008Splinter
001CP1DP2SP4D=NYYYYYY008Splinter
001CP1DP2SP4H=NYYYYYY008To play
001CP1DP2N=NYYYYYY016+ hcp, 6+!c. 18+ if 3!h
001CP1DP2NP3C=NYYYYYY008Suggestion to play
001CP1DP2NP3D=NYYYYYY008Relay
001CP1DP2NP3DP3H=NYYYYYY0083!h, 18+ hcp
001CP1DP2NP3H=NYYYYYY008Game forcing with 6+!h
001CP1DP3C=NYYYYYY00815--17 hcp, 6+!c and 3!h
001CP1DP3CP3D=NYYYYYY008Retransfer
001CP1DP3CP3H=NYYYYYY008Invitational
001CP1DP3D=NYYYYYY00817--19 bal, 4!h
001CP1DP3DP3H=NYYYYYY008To play
001CP1DP3H=NYYYYYY00813--15 hcp, good hand, 5+!c and 4!h
001CP1DP3HP3N=NYYYYYY0Asking for singleton
