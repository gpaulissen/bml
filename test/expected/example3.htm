<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>No title supplied</title><style type="text/css">div.start {
    display: inline-block;
    vertical-align: top;
    width: 60px;
}
    
div.bidtable {
    width:700px;
    padding-bottom: 20px;
}    

div.bidtable ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
}

div.bidtable li ul
{
    list-style-type: none;
    margin: 0;
    padding: 0;
    margin-left: 60px;
}

div.tree {
    list-style-type: none;
    padding-bottom: 20px;
}

div.tree ul {
    list-style-type: none;
    /* vline.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAKAQMAAABPHKYJAAAAA1BMVEWIiIhYZW6zAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1ggGExMZBky19AAAAAtJREFUCNdjYMAEAAAUAAHlhrBKAAAAAElFTkSuQmCC) repeat-y;
    margin: 0;
    padding: 0;
    margin-left: 10px;
}

div.tree li.node {
    margin: 0;
    padding: 0 12px;
    line-height: 20px;
    /* node.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIJYVaFGwAAAARSURBVAjXY2hgQIf/GTDFGgDSkwqATqpCHAAAAABJRU5ErkJggg==) no-repeat;
}

div.tree td.node {
    vertical-align: top;
    width: 60px;
}

div.tree li.last {
    /* lastnode.png */
    background: #fff url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIIhs+gc8AAAAQSURBVAjXY2hgQIf/GbAAAKCTBYBUjWvCAAAAAElFTkSuQmCC) no-repeat;
}

div.tree li.root {
    background: #fff none no-repeat;
}

.ccolor{color:Green}
.dcolor{color:Orange}
.hcolor{color:Red}
.scolor{color:Black}
</style></head><body><div class="bidtable"><ul><li><div class="start">1<span class="ccolor">&clubs;</span>-1<span class="dcolor">&diams;</span></div><ul><li><div class="start">1<span class="hcolor">&hearts;</span></div>Minimum with 2&ndash;3<span class="hcolor">&hearts;</span><ul><li><div class="start">1<span class="scolor">&spades;</span></div>4+<span class="hcolor">&hearts;</span>, 4<span class="scolor">&spades;</span>, at most invitational</li><li><div class="start">1NT</div>Sign off</li><li><div class="start">2<span class="ccolor">&clubs;</span></div>Puppet to 2<span class="dcolor">&diams;</span><ul><li><div class="start">2<span class="dcolor">&diams;</span></div>Forced<ul><li><div class="start">2<span class="hcolor">&hearts;</span></div>Mildly invitational with 5<span class="hcolor">&hearts;</span></li><li><div class="start">2<span class="scolor">&spades;</span></div>Invitational, 5+<span class="hcolor">&hearts;</span> and 4<span class="scolor">&spades;</span></li><li><div class="start">2NT</div>Strongly invitational with 5<span class="hcolor">&hearts;</span></li><li><div class="start">3<span class="ccolor">&clubs;</span></div>Invitational with 4<span class="hcolor">&hearts;</span> and 5+<span class="ccolor">&clubs;</span></li><li><div class="start">3<span class="dcolor">&diams;</span></div>Invitational with 4<span class="hcolor">&hearts;</span> and 5+<span class="dcolor">&diams;</span></li><li><div class="start">3<span class="hcolor">&hearts;</span></div>6<span class="hcolor">&hearts;</span>, about 11&ndash;12 hcp</li></ul></li></ul></li><li><div class="start">2<span class="dcolor">&diams;</span></div>Artificial game force</li><li><div class="start">2<span class="hcolor">&hearts;</span></div>6+<span class="hcolor">&hearts;</span>, about 9&ndash;10 hcp</li></ul></li><li><div class="start">1<span class="scolor">&spades;</span></div>5+<span class="ccolor">&clubs;</span>, 4+<span class="scolor">&spades;</span>, unlimited</li><li><div class="start">1NT</div>17&ndash;19 bal, 2&ndash;3<span class="hcolor">&hearts;</span></li><li><div class="start">2<span class="ccolor">&clubs;</span></div>5+<span class="ccolor">&clubs;</span>, unbal, 0&ndash;2<span class="hcolor">&hearts;</span>, 0&ndash;3<span class="scolor">&spades;</span></li><li><div class="start">2<span class="dcolor">&diams;</span></div>Reverse</li><li><div class="start">2<span class="hcolor">&hearts;</span></div>Minimum, 4<span class="hcolor">&hearts;</span></li><li><div class="start">2<span class="scolor">&spades;</span></div>16+ hcp, 5+<span class="ccolor">&clubs;</span> and 4+<span class="hcolor">&hearts;</span><ul><li><div class="start">3<span class="dcolor">&diams;</span></div>Retransfer<ul><li><div class="start">3<span class="hcolor">&hearts;</span></div><ul><li><div class="start">3<span class="scolor">&spades;</span></div>Cue bid, slam interest</li><li><div class="start">4<span class="ccolor">&clubs;</span><span class="dcolor">&diams;</span></div>Cue bid, slam interest</li><li><div class="start">4<span class="hcolor">&hearts;</span></div>To play</li></ul></li></ul></li><li><div class="start">3<span class="hcolor">&hearts;</span></div>Invitational</li><li><div class="start">3<span class="scolor">&spades;</span></div>Splinter</li><li><div class="start">4<span class="ccolor">&clubs;</span><span class="dcolor">&diams;</span></div>Splinter</li><li><div class="start">4<span class="hcolor">&hearts;</span></div>To play</li></ul></li><li><div class="start">2NT</div>16+ hcp, 6+<span class="ccolor">&clubs;</span>. 18+ if 3<span class="hcolor">&hearts;</span><ul><li><div class="start">3<span class="ccolor">&clubs;</span></div>Suggestion to play</li><li><div class="start">3<span class="dcolor">&diams;</span></div>Relay<ul><li><div class="start">3<span class="hcolor">&hearts;</span></div>3<span class="hcolor">&hearts;</span>, 18+ hcp</li></ul></li><li><div class="start">3<span class="hcolor">&hearts;</span></div>Game forcing with 6+<span class="hcolor">&hearts;</span></li></ul></li><li><div class="start">3<span class="ccolor">&clubs;</span></div>15&ndash;17 hcp, 6+<span class="ccolor">&clubs;</span> and 3<span class="hcolor">&hearts;</span><ul><li><div class="start">3<span class="dcolor">&diams;</span></div>Retransfer</li><li><div class="start">3<span class="hcolor">&hearts;</span></div>Invitational</li></ul></li><li><div class="start">3<span class="dcolor">&diams;</span></div>17&ndash;19 bal, 4<span class="hcolor">&hearts;</span><ul><li><div class="start">3<span class="hcolor">&hearts;</span></div>To play</li></ul></li><li><div class="start">3<span class="hcolor">&hearts;</span></div>13&ndash;15 hcp, good hand, 5+<span class="ccolor">&clubs;</span> and 4<span class="hcolor">&hearts;</span><ul><li><div class="start">3NT</div>Asking for singleton</li></ul></li></ul></li></ul></div></body></html>